### PR TITLE
Not all fields should be full width#188

### PR DESCRIPTION
**Pull request by razinabrahamyan**

### CODE REVIEW

**Review by alexivanenko — DISMISSED** [review]
Looks good to me

File: packages/oceanfront/src/scss/_fields.scss [hunk 1]
```diff
@@ -47,6 +47,9 @@
 .of--cursor-text {
   cursor: text;
 }
+.of--block{
+  width: 100%;
```
Comment (andrewwhitehead) [inline packages/oceanfront/src/scss/_fields.scss:51]: Maybe this should set `--of-field-width` to 100%?

**Comment by andrewwhitehead** [comment]
It seems like `size` is doing double duty at the moment, which is confusing. I think we should rename the old `size` property to `scale` (sm, md, lg, etc) and keep that behaviour: it changes the text size and the padding, not the width specifically. @alexivanenko ?

**Comment by razinabrahamyan** [comment]
@alexivanenko right the 'size' property doesn't affect the width, I agree with you, will update and push changes quickly

**Comment by razinabrahamyan** [comment]
@andrewwhitehead @alexivanenko There are other components that have size and changing font size example buttons and badges, should I change all to scale?

**Comment by alexivanenko** [comment]
> @andrewwhitehead @alexivanenko There are other components that have size and changing font size example buttons and badges, should I change all to scale?

@razinabrahamyan I think yes for consistency. But we need to update app-js after this.

**Comment by razinabrahamyan** [comment]
@alexivanenko ok, got it

**Review by andrewwhitehead — APPROVED** [review]

**Pull request merged**